src/core/shape_inference/include/fake_convert_shape_inference.hpp

```diff
@@ -13,16 +13,17 @@ template <class T, class TRShape = result_shape_t<T>>
 std::vector<TRShape> shape_infer(const FakeConvert* op, const std::vector<T>& input_shapes) {
     const auto inputs_count = input_shapes.size();
     const auto has_shifts_input = inputs_count == 3;
-    NODE_VALIDATION_CHECK(op, inputs_count  == 2 || has_shifts_input );
+    NODE_VALIDATION_CHECK(op, inputs_count == 2 || has_shifts_input);
     if (has_shifts_input) {
-        NODE_SHAPE_INFER_CHECK(op, input_shapes, input_shapes[1].compatible(input_shapes[2]),
-                        "FakeConvert scale shape is not compatible with shift shape.");
+        NODE_SHAPE_INFER_CHECK(op,
+                               input_shapes,
+                               input_shapes[1].compatible(input_shapes[2]),
+                               "FakeConvert scale shape is not compatible with shift shape.");
     }
     TRShape output_pshape = input_shapes[0];
-    NODE_VALIDATION_CHECK(
-        op,
-        TRShape::broadcast_merge_into(output_pshape, input_shapes[1], op::AutoBroadcastType::NUMPY),
-        "Argument shapes are inconsistent.");
+    NODE_VALIDATION_CHECK(op,
```

Comment:
Use, `NODE_SHAPE_INFER_CHECK` it should be in most cases default macro in shape inference except the inputs size check, or when the input shapes have no meaning in error message.

Reply:
Done